library/src/window_impl.cxx

```diff
@@ -245,14 +230,6 @@ window_impl::window_impl(const options& options, const std::optional<Type>& type
     {
       oglRenWin->SetOpenGLSymbolLoader(&internals::SymbolLoader, &this->Internals->GetProcAddress);
     }
-#if F3D_MODULE_EXTERNAL_RENDERING
-    if (oglRenWin->IsA("vtkExternalOpenGLRenderWindow"))
-    {
-      // We need to call vtkOpenGLRenderWindow function because vtkGenericOpenGLRenderWindow is
-      // reimplementing it incorrectly
```

Comment:
does that work out of the box with your window ?

Reply:
It is fixed with my `Render()` override which call `OpenGLInit` from `vtkOpenGLRenderWindow`

Reply:
nice